reporter/src/main/kotlin/reporters/AsciiDocTemplateReporter.kt

```diff
@@ -100,6 +101,13 @@ class AsciiDocTemplateReporter : Reporter {
 
                 asciidoctorAttributes.attribute("pdf-theme", themePath)
             }
+            templateOptions.remove(OPTION_PDF_FONTS_DIR)?.let { fontsPath ->
+                File(fontsPath).also {
+                    require(it.isDirectory) { "Could not find pdf-fontsdir path at '${it.absolutePath}'." }
+                }
+
+                asciidoctorAttributes.attribute(OPTION_PDF_FONTS_DIR, "${File(fontsPath).absolutePath},GEM_FONTS_DIR")
```

Comment:
We already constructed `File(fontsPath)` above and should reuse it here by introducing a variable.

Reply:
But can't we just pass the (possibly relative) `fontPath` here? If not, shouldn't we than also make the existing `themePath` use ab absolute path?